llama_stack/distribution/store/registry.py

```diff
@@ -13,11 +13,8 @@
 from llama_stack.distribution.datatypes import KVStoreConfig, RoutableObjectWithProvider
 from llama_stack.distribution.utils.config_dirs import DISTRIBS_BASE_DIR
 
-from llama_stack.providers.utils.kvstore import (
-    KVStore,
-    kvstore_impl,
-    SqliteKVStoreConfig,
-)
+from llama_stack.providers.utils.kvstore import KVStore, kvstore_impl
+from llama_stack.providers.utils.kvstore.config import SqliteKVStoreConfig
```

Comment:
how do we support other KVStores if we are only importing Sqlite's config?

Reply:
I think this file only uses SqliteKVStoreConfig as default if other kvstore config is not passed to `create_dist_registry` cc @dineshyv to confirm. 

https://github.com/meta-llama/llama-stack/blob/70db039ff4dfa486a959243764725226291ac5ee/llama_stack/distribution/store/registry.py#L195-L206